Formula/cmake.rb

```diff
@@ -12,10 +12,10 @@ class Cmake < Formula
     sha256 "a940f8d8e9818512c466ffd205612cb1b477d936e87ab3d6c7dac888f6d03b92" => :sierra
   end
 
-  option "with-completion", "Install Bash completion (Has potential problems with system bash)"
-
   depends_on "sphinx-doc" => :build
 
+  # The completions were removed becuase of problems with system bash
```

Comment:
because

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/commit/52ddac7b8d007f07a2a9bdd8c0cd814bda16def8